app/docs/enterprise/0.31-x/oidc-auth0.md

```diff
@@ -0,0 +1,41 @@
+---
+title: OpenID Connect with Auth0
+class: page-install-method
+---
+
+This guide covers an example OpenID Connect plugin configuration to authenticate headless services using Auth0's identity provider.
+
+# Auth0 IDP configuration
+
+This configuration will use a [client credentials grant](https://auth0.com/docs/api-auth/tutorials/client-credentials) as it is non-interactive, and because we expect clients to authenticate on behalf of themselves, not an end-user. To do so, you will need to [create an Auth0 API](https://auth0.com/docs/apis#how-to-configure-an-api-in-auth0) and a [non-interactive client](https://auth0.com/docs/clients).
+
+## API configuration
+
+When creating your API, you will need to specify an Identifier. Using the URL that your service makes requests to is generally appropriate, so this will typically be the hostname/path combination configured as a service or API in Kong.
```

Comment:
"Using the URL that your service makes requests to.." should "service" be "client" or perhaps "consumer"?

"this will typically be the hostname/path combination configured as a service or API in Kong" - because Kong EE 0.31 supports only APIs, not Routes and Services, and because this doc is versioned, this should say only "...as an API in Kong" and not refer to service.

Reply:
There's unfortunately overlap between the terms used by Kong and the OIDC spec. Client probably makes more sense in general usage, but refers to something else in OIDC terms. Consumer more or less works, although the consumer in this example isn't configured in Kong

Reply:
This sort of thing should be clarified by a "Terminology" section as many other plugins and guides already do on this website.